site/en/docs/webstore/faq/index.md

```diff
@@ -193,11 +193,13 @@ quickly.
 
 ### How can I remove my listing from the store? {: #faq-listing-03 }
 
-Visit [https://chrome.google.com/webstore/developer/dashboard][115]. Click the item you want to
+Visit the [developer dashboard][115]. Click the item you want to
```

Comment:
115? The links in this file are out of control. Can you please file a bug to reconcile the links in this file later? Thanks!

Reply:
As a side note @AmySteam, we don't like the numbered [115] links—this is just an artifact of the way this content was imported. So we definitely want to clean them up to be named or inline (I actually don't have a preference).

Reply:
Yeah that makes sense. I'll start converting the links I find from now on. Thanks for the tip 🙏🙂🌻